MODULE.bazel

```diff
@@ -764,12 +764,17 @@ dev_maven.install(
         "androidx.annotation:annotation:1.6.0",
         # https://github.com/bazel-contrib/rules_jvm_external/issues/1409
         "com.squareup.okhttp3:okhttp:4.12.0",
+        # Versioned snapshot pinning support: https://github.com/bazel-contrib/rules_jvm_external/pull/1412
+        "com.google.guava:guava:999.0.0-HEAD-jre-SNAPSHOT",
+        # Non-versioned snapshot pinning support: https://github.com/bazel-contrib/rules_jvm_external/pull/1412
+        "org.seleniumhq.selenium:selenium-java:4.34.0-SNAPSHOT",
```

Comment:
You'll need to port this change here as well https://github.com/bazel-contrib/rules_jvm_external/blob/master/WORKSPACE#L390 in WORKSPACE to resolve the CI failures related to Bazel 6.

Reply:
Thanks added